lib/resources/r4/related_person_patch.yaml

```diff
@@ -98,3 +99,25 @@ operations:
         <li>Requires a <a href="#test-name-id">test operation</a> to be provided for the `HumanName.id` of the name whose `family` is intended to be replaced.</li>
         <li>This operation is only supported for the first name in the list of related person names (<code>{index}</code> == 0).</li>
       </ul>
+
+  - name: replace-name-given
+    path: /name/{index}/given
+    operation: replace
+    type: String
+    url: https://hl7.org/fhir/R4/datatypes-definitions.html#HumanName.given
+    description: Replace the `HumanName.given` of the name at the given <code>{index}</code> in the list of related person names.
+    example: |
+      {
+        "path": "/name/0/given",
+        "op": "replace",
+        "value": [
+          "Harry",
+          "James"
```

Comment:
Let's make the examples have the same data.

Reply:
Changed : https://github.com/cerner/fhir.cerner.com/commit/c28fd706c919ca8c839c0ecb332e8d9d04a527f1#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR821